openapi/resource-server.yaml

```diff
@@ -32,8 +32,8 @@ info:
   contact:
     email: tech@interledger.org
 servers:
-  - url: '/'
-    description: 'Server for wallet address subresources or Connection resources (ie https://openpayments.guide/alice)'
+  - url: 'https://openpayments/'
```

Comment:
Not sure if I have a preference, but `rafiki.money` might be nice to have if we are going to include some code snippets in the documentation that _do_ use `rafiki.money`. I would connect with Radu /Melissa/Mohammed for this

Reply:
I will change to rafiki.money. Melissa confirmed that there will be code snippets in the docs that are intended to be used with rafiki.money. https://github.com/interledger/open-payments-snippets She also noted the openpayments.guide domain might change as well but in any case it seems like the rafiki.money url will be more useful.